src/VariantViewer.js

```diff
@@ -31,10 +31,23 @@ var variantsFill = function(d, fv) {
         } else {
             return LegendDialog.UPNonDiseaseColor;
         }
-    } else if (d.siftScore !== undefined) {
-        return LegendDialog.getPredictionColor((d.siftScore + (1-d.polyphenScore))/2);
     } else {
-        return LegendDialog.othersColor;
+        var sift = false, polyphen = false;
```

Comment:
I think this could be done in less lines of code. You don't actually use polyphenPrediction/siftPrediction just siftScore/polyphenScore so should be able to check they exist directly in the conditionals.